### ISSUE TITLE
rtdl: `RTLD_LOCAL` and `RTLD_LAZY` have the same value

### ISSUE THREAD

**Issue by 64** [managarm/mlibc#450]

**Comment by 64** [comment]
I don't think this is actually a correctness issue at present, since

1. RTLD_LOCAL is zero in other libcs, so if an application needs RTLD_GLOBAL it will already specify it
2. RTLD_LAZY is just a hint to the implementation so it shouldn't cause problems (given that we ignore it currently)

However it should still probably be fixed.

**Comment by 64** [comment]
> I think we can change RTLD_LOCAL to be zero without a major release.

RTLD_LOCAL is already zero (they both are)

**Comment by avdgrinten** [comment]
Ah. That should also be fine without a major release since it won't affect the behavior of our dlopen() anyway.

**Comment by 64** [comment]
Agreed. I have fixed this in d6e1a681b981fc912b6ab4774fcbd884282aec05.
